src/components/License/VLicense.vue

```diff
@@ -43,6 +41,8 @@ import nd from '~/assets/licenses/nd.svg'
 import pdm from '~/assets/licenses/pdm.svg'
 import sa from '~/assets/licenses/sa.svg'
 
+import VIcon from '~/components/VIcon/VIcon.vue'
```

Comment:
Hmmm, I'd like components to come _before_ assets. I'm not sure why this one is after them :confused: Maybe my partial matching approach isn't working as expected.

Reply:
Fixed in https://github.com/WordPress/openverse-frontend/pull/999/commits/66f9cd3f42f7767fcae5948072fe24ef94332651 . Indeed, I have no idea how minimatch's `partial` option is meant to work :shrug: